spec/classes/collectd_init_spec.rb

```diff
@@ -30,7 +30,7 @@
    end
 
    it 'should fail' do
-     expect { subject }.to  raise_error(/foo is not supported/)
+     should compile.and_raise_error(/foo is not supported./)
```

Comment:
lousy copy-paste. tests are broken anyway for other reasons in travis... i'm on it right now